WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -317,6 +314,26 @@ private extension SettingsViewController {
         StoresManager.shared.deauthenticate()
         navigationController?.popToRootViewController(animated: true)
     }
+
+    func weAreHiringWasPressed(url: URL) {
+        WooAnalytics.shared.track(.settingsWereHiringTapped)
+
+        let safariViewController = SFSafariViewController(url: url)
```

Comment:
We have a `WebviewHelper` that provides this behaviour as well, would you consider switching to it?